src/actions/userActions.js

```diff
@@ -64,6 +64,11 @@ export function authenticateUser(userCreds) {
         }).then((response) => {
             return response.json();
         }).then((json) => {
+            console.log(json);
+            if (!json.success) {
```

Comment:
This change got in but isn't related to the PR, but our json responses can also error, and the `.catch` block doesn't catch it. However, `json.success` will be true if there's no error.